src/lib/TSL/Splitter.hs

```diff
@@ -124,13 +125,32 @@ split spec@Specification{assumptions, guarantees} =
     connComp = map Set.fromList $ components graph
     (freeAssumptions, boundAssumptions) = partition (null . (intersection decRelProps) . getInOutputs) assumptions
     splitGuars = splitFormulas guarantees connComp
```

Comment:
I would patch this variable naming as well, if you choose to fix for assumptions.

Reply:
done in ba2a725